tests/python/contrib/test_arm_compute_lib/test_network.py

```diff
@@ -180,7 +183,12 @@ def get_model():
         return mod, params, inputs
 
     _build_and_run_network(
-        *get_model(), device=device, tvm_ops=3, acl_partitions=30, atol=9, rtol=0
+        *get_model(),
+        device=device,
+        tvm_ops=3,
+        acl_partitions=30,
+        atol=9,
+        rtol=0.01,
```

Comment:
Also curious why `rtol` changed here?

Reply:
I'm guessing as we now run slightly different schedules we get slightly different answers, this aligns with the above `rtol` though.

Reply:
Hmm, seems a bit odd to me since the output for this particular model should be uint8

Reply:
Hmm seems a bit odd for sure, could be due to a difference in the random input values. I think it makes more sense to adjust the `atol` in this case

Reply:
@lhutton1 came to the same conclusion, I've upped it by `1` to see if that makes it happier 😸